tests/aws-spawn-test.sh

```diff
@@ -60,3 +60,4 @@ done
 for version in 16-04 18-04 20-04 22-04; do
   test ubuntu "$version"
 done
+test alpine
```

Comment:
I only see this now, but it's not a very good idea to hide the `test` utility this way.